main/vstring.c

```diff
@@ -81,12 +81,12 @@ extern vString *vStringNew (void)
 
 extern void vStringPut (vString *const string, const int c)
```

Comment:
@codebrainz, thank you. You know C and these areas well. This is not the first time helping me with your knowledges of the area. Could you help the development of u-ctags explicitly? I sent an invitation.